main.go

```diff
@@ -44,14 +47,25 @@ func main() {
 		os.Exit(1)
 	}
 
-	broker := broker.New(config, newCatalog, &logger)
+	compose, err := compose.NewClient(config.APIToken)
+	if err != nil {
+		logger.Error("could not create composeapi client", err)
+		os.Exit(1)
+	}
+
+	b, err := broker.New(compose, config, &newCatalog, logger)
```

Comment:
It's problematic to have a variable named the same as an imported package. I agree this could do with a clearer name though.